utils/coreutils/techutils.go

```diff
@@ -16,6 +16,7 @@ const (
 	Go     = "go"
 	Pip    = "pip"
 	Pipenv = "pipenv"
+	Poetry = "poetry"
```

Comment:
@omerzi created a PR which unifies the package managers name across the entire code. Worth synching with him to make the merge easier.